fblldbbase.py

```diff
@@ -36,14 +36,27 @@ def run(self, arguments, option):
     pass
 
 
-def evaluateExpressionValue(expression, printErrors=True):
+def evaluateExpressionValueWithLanguage(expression, language, printErrors):
   # lldb.frame is supposed to contain the right frame, but it doesnt :/ so do the dance
   frame = lldb.debugger.GetSelectedTarget().GetProcess().GetSelectedThread().GetSelectedFrame()
-  value = frame.EvaluateExpression(expression)
+  expr_options = lldb.SBExpressionOptions()
+  expr_options.SetLanguage(language)  # requires lldb r210874 (2014-06-13) / Xcode 6
```

Comment:
Hey @arigrant, what do you think about baseline support? Should Chisel support anything more than just the current version of Xcode?